app/Http/Controllers/DeviceController.php

```diff
@@ -604,8 +607,19 @@ public function ajaxCreate(Request $request)
             $device[$i]->parts_provider = $parts_provider;
             $device[$i]->event = $event_id;
             $device[$i]->repaired_by = Auth::id();
+
             $device[$i]->save();
 
+            if ($useful_url && $useful_source) {
+                // Devices can have multiple URLs, but we only support one on the create - and it gets applied to each
```

Comment:
As discussed.

So at present additional ones can be added only on edit.  We will revisit this in future sprint.

Reply:
How should we ensure that happens?

Reply:
I've added a story to Jira - https://therestartproject.atlassian.net/browse/DOT-1305